ci/dockerfile/ubuntu18.04.dockerfile

```diff
@@ -1,6 +1,14 @@
 # Start with 18.04
 FROM ubuntu:18.04
 
+RUN apt update && DEBIAN_FRONTEND=noninteractive apt install -y curl gnupg2 binutils
```

Comment:
Can we put this workload specific installation in Makefile?

Reply:
I will make the change